packages/components/buttons/PrimaryButton.tsx

```diff
@@ -114,7 +116,7 @@ export const PrimaryButton: React.FC<{
             style={[
               fontSemibold14,
               {
-                color: primaryTextColor,
+                color: theme.secondaryTextColor,
```

Comment:
Because in the UI design, this button don't use `primaryTextColor` for `lightTheme` 👍 
We'll have a call today to discuss about future refacto phases, and i want to discuss about relation between design and code 👍